keras/callbacks.py

```diff
@@ -858,44 +858,30 @@ def is_indexed_slices(grad):
 
             projector.visualize_embeddings(self.writer, config)
 
+    def _fetch_callback(self, summary):
+        self.writer.add_summary(summary, self._epoch)
```

Comment:
I believe there's a problem here: we would be writing a summary at every batch, but every summary has the same name. That would lead to display errors. What do you see in TensorBoard?

Reply:
I tested this, and only a single batch was used.
I adjusted the implementation as described [here](https://github.com/keras-team/keras/pull/10513#issuecomment-404392571).